pages/index.tsx

```diff
@@ -1,24 +1,32 @@
 import * as React from 'react'
-import Curriculum from './curriculum'
+import _ from 'lodash'
+import { useRouter } from 'next/router'
 import AppNav from '../components/AppNav'
 import LandingPage from '../components/LandingPage'
 import Footer from '../components/Footer'
 import SessionContext from '../helpers/contexts/session'
 
 const IndexPage: React.FC = () => {
+  const router = useRouter()
   const { session } = React.useContext(SessionContext)
+  const username = _.get(session, 'user.username')
 
-  if (session) {
-    return <Curriculum />
+  if (username) {
+    router.push('/curriculum')
+    return null
   }
 
-  return (
-    <>
-      <AppNav loggedIn={false} />
-      <LandingPage />
-      <Footer footerType="py-5 bg-white text-muted" />
-    </>
-  )
+  if (session)
```

Comment:
do you need this `if` condition? Why not just return directly?

Reply:
Yes. Otherwise React Testing Library will throw an error for the case where there's no session.